src/functions-templates/javascript/scheduled-function/{{name}}.js

```diff
@@ -0,0 +1,18 @@
+const { schedule } = require('@netlify/functions')
+
+// Sample function scheduled to run every minute using conventional cron syntax.
```

Comment:
This comment is great info, but I worry that people will feel intimidated by the amount of information in it. People shouldn't need to learn cron expressions to make use of this feature, which is why I suggested in a previous review that we defaulted to something like `@hourly`.

To accommodate more advanced use cases, I would add a comment saying that we support any cron expression and include a link to something like https://crontab.guru/ to allow people to create one, or familiarise themselves with the concept.

Reply:
I remember some discussion around what cron explainer service we should point to here. maybe it's time to fork crontab.guru? :D

Reply:
I agree with defaulting the cron expression to @hourly, which eliminates the noisy comments in the code. Additionally, I think it makes more sense to refer users to the Labs documentation about [Scheduled Functions](https://ntl.fyi/sched-func) as it feels better adapted to explain what they're about and provides info about supported cron extensions.

Reply:
@karagulamos Agree! I think a link to https://ntl.fyi/sched-func is all we need.